examples/shader/post_processing.rs

```diff
@@ -338,6 +338,8 @@ impl FromWorld for PostProcessPipeline {
 #[derive(Component, Default, Clone, Copy, ExtractComponent, ShaderType)]
 struct PostProcessSettings {
     intensity: f32,
+    #[cfg(feature = "webgl2")]
+    _webgl2_padding: Vec3,
```

Comment:
There should be a comment explaining why this is needed

Reply:
Duplicated the comment form the shader side here.